src/features/user/Profile/components/RedeemModal.tsx

```diff
@@ -266,17 +273,25 @@ export default function RedeemModal({
           <div className={styles.modal}>
             {codeValidated && validCodeData ? (
               <>
-                <div className={styles.codeTreeCount}>
+                <div
+                  className={styles.codeTreeCount}
+                  style={{ fontSize: '2rem' }}
+                >
                   {getFormattedNumber(
                     i18n.language,
-                    Number(validCodeData.treeCount)
+                    Number(validCodeData.units)
                   )}
                   <span>
                     {t('common:tree', {
-                      count: Number(validCodeData.treeCount),
+                      count: Number(validCodeData.units),
                     })}
                   </span>
                 </div>
+                <div>
+                  <span className={styles.codeTreeCount}>
+                    {t('redeem:successfullyRedeemed')}
+                  </span>
+                </div>
```

Comment:
It is a little confusing to use `styles.codeTreeCount` here, as that suggests that the data shown here is tree count. Either use a more general classname, or create a separate class.